OpenEdXMobile/src/main/java/org/edx/mobile/module/registration/view/RegistrationTextAreaView.java

```diff
@@ -9,18 +9,19 @@
 class RegistrationTextAreaView extends RegistrationEditTextView {
 
     // Number of lines for TextArea
-    private static final int LINES = 5;
+    private static final int INIT_LINES = 1;
+    private static final int MAX_LINES = 5;
```

Comment:
Now that the font is significantly bigger in textinputlayout, we should also increase the max lines.